docs/getting-started/zowe-secure-setup.md

```diff
@@ -77,15 +77,15 @@ TLS_AES_128_GCM_SHA256, TLS_AES_256_GCM_SHA384, TLS_CHACHA_POLY1305_SHA256
 For more information, see the [TLS requirements in Zowe API ML requirements](../extend/extend-apiml/zowe-api-mediation-layer-security-overview#zowe-api-ml-tls-requirements).
 
 ### Digital certificates types
-The digital certificates can be issued in various formats. The format is dependent on the certificate storage type PKCS12 or JKS/JCEKS.
-The most general and widely deployed certificate format today is PKCS12, while the JKS and the more extended JCEKS are specific for Java environments.
+Digital certificates can be issued in various formats. The format is dependent on the certificate storage type PKCS12 or JKS/JCEKS.
+The most general and widely deployed certificate format today is PKCS12, while the JKS and the more extended JCEKS are specific for depending on the Java environment.
 
-**Note: Java 9 and higher can also work with PKCS12 certificates
+**Note:** Java 9 and higher can also work with PKCS12 certificates.
 
 ### Certificates storage
 
 - Keystores and truststores
```

Comment:
I'm wondering if this would be a good place to add the section Choosing USS Keystore or z/OS Key Ring to store Zowe certificates: See https://github.com/zowe/docs-site/blob/cf13166ed2eb98f9f69bfc5c65a7298ee174cdf8/docs/user-guide/configure-certificates.md#choosing-uss-keystore-or-zos-key-ring-to-store-zowe-certificates

Reply:
I'd suggest moving this detailed note to the appendix with the PKI/TLS definitions or to the certificates configuration document, where the user will be in the context of a concrete setup.